fictadvisor-api/src/v2/api/controllers/DisciplineTeacherController.ts

```diff
@@ -307,4 +309,59 @@ export class DisciplineTeacherController {
     const updatedComment = await this.disciplineTeacherService.updateComment(disciplineTeacherId, body);
     return this.questionMapper.getUpdatedComment(updatedComment);
   }
+
+  @ApiBearerAuth()
+  @ApiOkResponse()
+  @ApiBadRequestResponse({
+    description: `\n
+    InvalidEntityIdException:
+      User with such id is not found
+      Question with such id is not found
+      DisciplineTeacher with such id is not found
+      
+    InvalidTypeException
+      Question has wrong type`,
+  })
+  @ApiForbiddenResponse({
+    description: `
+      NoPermissionException: You do not have permission to perform this action
+    `,
+  })
+  @ApiUnauthorizedResponse({
+    description: `\n
+    UnauthorizedException:
+      Unauthorized`,
+  })
+  @ApiForbiddenResponse({
+    description: `\n
+    NoPermissionException:
+      You do not have permission to perform this action`,
+  })
+  @ApiParam({
+    name: 'questionId',
+    required: true,
+    description: 'Question Id',
+  })
+  @ApiParam({
+    name: 'disciplineTeacherId',
+    required: true,
+    description: 'Discipline teacher id',
+  })
+  @ApiParam({
+    name: 'userId',
+    required: true,
+    description: 'User id',
+  })
+  @ApiEndpoint({
+    summary: 'Delete question answers with TEXT type (comments)',
+    permissions: PERMISSION.COMMENTS_DELETE,
+  })
+  @Delete('/:questionId/:disciplineTeacherId/:userId/comment')
```

Comment:
You don't have a conceptual understading what is a controller. A Controller is a bunch of endpoints which interacts with a specific entity/domain/field of project/sphere, etc. In this case, this controller intented to work with DisciplineTeacher entity. Thus, all endpoints should be connected to the discipline teacher. 

At the same time, your endpoint starts with ":questionId" param. It's not correct. It should start with ":disciplineTeacher". You can see the example at the endpoint above that edits comment. You need to have "/:disciplineTeacherId/comment" path and then use body object to define what comment should be deleted